index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css">
+  <title>Yuriy's Portfolio</title>
+</head>
+
+<body>
+  <div id="container">
+    <nav id="navigation">
+      <span id="logo">My Logo</span>
+      <ul id="menu">
+        <li class="menu-item big">Portfolio</li>
+        <li class="menu-item big">About</li>
+        <li class="menu-item big">Contact</li>
+        <li class="menu-item" id="hidden"><i class="fas fa-bars"></i></li>
+      </ul>
+    </nav>
+    <section id="bottom">
+      <header id="main">
+        <h2 class="title">I'm Yuriy</h2>
+        <h2 class="title">Glad to see you!</h2>
+        <p id="main-text">I’m a Full Stack Software Developer! I can help you build a product, feature or website. Look
+          through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <span id="connect-text">LET'S CONNECT</span>
+        <div id="social-icon">
+          <i class="fab fa-twitter"></i>
+          <i class="fab fa-linkedin-in"></i>
+          <i class="fab fa-github"></i>
+          <i class="fab fa-medium-m"></i>
+          <i class="fab fa-angellist"></i>
+        </div>
```

Comment:
- [ ] Since this is a list of icons, I think is best if you wrap them in a `ul` tag and also wrap each one of them in a `li` tag, this will be more [semantically adequate](https://www.w3schools.com/html/html5_semantic_elements.asp).